test-menus.py

```diff
@@ -0,0 +1,94 @@
+import unittest
```

Comment:
Please rename the file. A dash in a filename is considered very bad practice.

Reply:
Done.